iocore/net/SSLUtils.cc

```diff
@@ -141,8 +142,9 @@ static int ssl_vc_index = -1;
 static ink_mutex *mutex_buf      = nullptr;
 static bool open_ssl_initialized = false;
 
-RecRawStatBlock *ssl_rsb                   = nullptr;
-static InkHashTable *ssl_cipher_name_table = nullptr;
+RecRawStatBlock *ssl_rsb = nullptr;
+typedef std::unordered_map<const char *, intptr_t> cipherTable;
```

Comment:
That's what I thought too (we provide custom hashing functions for this in a few places), but then I tested this on a modern compiler, and it seemed to do it right. I assume C++11 had added support for hashing the char*, but now that you say this, it's quite possible my tests were wrong.